src/components/sbb-timetable-occupancy-icon/sbb-timetable-occupancy-icon.e2e.ts

```diff
@@ -0,0 +1,17 @@
+import { assert, fixture } from '@open-wc/testing';
```

Comment:
would it be possible to simulate window matchMedia?

Reply:
I added a test for high contrast using sinon, but I have trouble redefining the stub to test the prefer-color-scheme, will check again tomorrow